docs/configuration.md

```diff
@@ -1184,7 +1184,10 @@ should have both preceding and trailing slashes, for example <code>/path/</code>
     <td>
 A random piece of data. This is used to sign state that may be stored with the
 client to protect against tampering. For production environments you should generate a
-random piece of data using a cryptographically secure random generator.
+random piece of data using a cryptographically secure random generator. This
+configuration parameter may be omitted, in which case notary will automatically
+generate a secret at launch. However, a secret should be specified in the
+configuration file if multiple registries are used with a load balancer.
```

Comment:
Make that very strong, impossible to overread, bold, red, ital, double body. Maybe something like (please fix my engrish):

***WARNING: if you are building a cluster of registries behind a load balancer, you MUST ensure the secret is the same for all registries hosts***.